lib/intersectKeys.js

```diff
@@ -0,0 +1,26 @@
+// ------------------------------------
+// #POSTHTML - CONTENT - INTERSECTKEYS
+// ------------------------------------
+
+'use strict'
```

Comment:
these are not needed either -- if you do wanna add docs probably [jsdoc](http://usejsdoc.org/) is best 👍